pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -261,6 +261,11 @@ class MindtPySolver(object):
         description="Add OA cuts for inactive constraints.",
         domain=bool
     ))
+    CONFIG.declare("use_mcpp", ConfigValue(
+        default=False,
+        description="use mcpp for to set bound for variable 'objective_value' introduced when the objective function is nonlinear.",
```

Comment:
We should clarify that the default uses fbbt to compute bounds.

Reply:
could we rephrase the description? I'm not sure I fully understand it

Reply:
Good point. I suggest changing the description to "use package MC++ to set a bound for variable 'objective_value', which is introduced when the original problem's objective function is nonlinear."